src/common/dts.c

```diff
@@ -316,8 +318,9 @@ cont_init(struct dts_context *tsc)
 	if (rc)
 		goto out; /* open failed */
```

Comment:
Why this call without "if (!tsc->tsc_pmem_file)"?

Reply:
Its not specific for non-vos_perf mode.